docs/system-admin-guide/manage-work-packages/work-package-status/README.md

```diff
@@ -12,24 +12,42 @@ Work packages can have various status depending on their types, e.g. tasks can h
 
 The status can be set at the top of the work package details view or in the table view in the corresponding column.
 
-![Work packages status dropdown menu in OpenProject](openproject_system_guide_create_wp_stati_dropdown.png)The status field can be configured under *Administration ->* *Work packages* -> *Status*. You will see the list of all the existing work package status in the system. You can add new status or change the name and order of existing status.
+![Work packages status dropdown menu in OpenProject](openproject_system_guide_create_wp_stati_dropdown.png)
+
+The status field can be configured under *Administration ->* *Work packages* -> *Status*. You will see the list of all the existing work package stati in the system. You can add new status or change the name and order of existing status.
+
+![Work package status overview in OpenProject administration](openproject_system_guide_create_wp_status_overview.png)
+
 To learn about the options for configuring the transition from one status to another navigate to [Manage work package workflows](../work-package-workflows).
 
 ## Create a new work package status
 
 To create a new work package status click the green ***+ Status*** icon.
 
+
 A new window will open, where you will be able to specify the following:
 
 1. Name the new work package status.
+
 2. The %&nbsp;Complete value in [Status-based progress calculation mode](../work-package-settings/).
+
+   > [!TIP]
+   >
+   > The value for % Complete can be set from 1 to 100. 
+   >
+   > A decimal value will be truncated, i.e. if you input 5.7, the value set will be 5.
```

Comment:
I don't think it allows decimals, actually. The field has form validations that will show an error and ask you to enter a valid value.

Reply:
Yes, you're right that's what the feature describes. But when implementing it, I used a number field and this gives it some automatic checks handled by the browser, meaning the browser will refuse any value which is not strictly an integer, and it will display a relevant validation error message when the "Save" button is clicked like so:

![image](https://github.com/user-attachments/assets/16fdb7ad-1259-4984-9e93-a2e5a780903a)

So Parimal is right. I think the whole paragraph can be removed. It's not worth mentioning that decimal values are not allowed.

Sorry for the confusion, I'll update the feature description.

Reply:
Thank you @cbliard